source/swap/test/Swap-unit.js

```diff
@@ -134,21 +138,27 @@ contract('Swap Unit Tests', async accounts => {
 
     it('test adding token that does not transfer swap incorrectly and transfer returns false', async () => {
       // create mocked contract to test transfer
-      const fungibleTokenTemplate = await FungibleToken.new()
+      const handlerTemplate = await ERC20TransferHandler.new()
       const tokenMock = await MockContract.new()
+      const TransferHandlerRegistryTemplate = await TransferHandlerRegistry.new()
+      const FungibleTokenTemplate = await FungibleToken.new()
 
-      const token_balance = fungibleTokenTemplate.contract.methods
-        .balanceOf(EMPTY_ADDRESS)
+      const getHandler = TransferHandlerRegistryTemplate.contract.methods
```

Comment:
to follow our naming convention, this should probably be `registry_ getTransferHandler`